scripts/build.sh

```diff
@@ -27,6 +27,8 @@ while [[ $# -gt 0 ]]; do
       ;;
     --ci)
       ci=true
+      export CC=clang-10
+      export CXX=clang++-10
```

Comment:
Is this going to break my clang 9 builds on Debian 11?

Reply:
No - with that said, you should _not_ be using the ci flag unless you're specifically trying to build a la how AzDO is doing it.

With that said, AzDO has multiple versions of clang installed by default... so clang 10 was defined here.
If you want, we can test just defining `clang` however we may run into issues.

Reply:
No that's fine, I'm not using CI, was just curious.